core/commonMain/src/Arguments.kt

```diff
@@ -95,12 +119,14 @@ class MultipleArgument<T : Any> internal constructor(descriptor: ArgDescriptor<T
 }
 
 /**
- * Allow argument have several values.
+ * Allows the argument to have several values specified in command line string.
  *
- * @param value number of arguments are expected.
+ * @param value the maximum number of arguments expected for this parameter, but at least 2.
  */
+// TODO: rename the parameter to 'number'?
```

Comment:
It was firstly called number, but there was a discussion that it's better to use `multiple`

Reply:
I mean the name of parameter (`value` -> `number`)

Reply:
Ok, I missed "parameter" word, sorry